utils/searxng.sh

```diff
@@ -688,7 +688,7 @@ To install uWSGI use::
         rst_para "\
 The configured redis DB is not available: If your server is public to the
 internet, you should setup a bot protection to block excessively bot queries.
-Bot protection requires a redis DB.  About bot protection visit the official
+Bot protection requires a redis DB. About bot protection visit the official
```

Comment:
The double space after the dot of sentence end was intended. Typographic tools use the two spaces after the end of a sentence to distinguish from the period of an abbreviation.

Reply:
Ah Okay, I'll change it now thank you